### DISCUSSION TITLE
[Technical Questions] XCode Preview and Build Simulator Different Behavior, Cut Off Descriptions

### DISCUSSION THREAD

**Discussion by bryant-jimenez** [CS342/.github#46] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

I am working on editing the contact view and adding our group's PIs contact information to the view. I have added followed the structure of the example application and created localized variable strings for the bios/description for each person's contact that I am editing and adding. In the preview for the application in xcode, I can see the entirety of the description just fine and the contact view behaves as it should, but when running the build, the ios simulator cuts off the description and doesn't allow to be able to see the rest of it. Is there any known fix for this or is it okay as long as it shows in the preview correctly? Thank you!

<img width="500" alt="image" src="https://github.com/CS342/.github/assets/84168212/b1cfd0bf-d192-4646-9d0e-cbac00e5860e">

### Reproduction

1. Preview in XCode shows the entire description within the contact
2. Run build 
3. When in the iOS build + simulator, open the app and go to the "Contacts" view. Cut off descriptions can be seen there subsequently

### Expected behavior

The description should extend and the contact view should be longer to incorporate the entire description, or there should be an interactable way for the user to expand the description and read it entirely

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Hi @bryant-jimenez,
Thank you for using GitHub Discussions to ask your question. It's a great fit! We are currently looking into the issue. It seems like this might be a bug in [Spezi Contacts](https://swiftpackageindex.com/StanfordSpezi/SpeziContact) or even the underlying [`Label` implementation in Spezi Views](https://swiftpackageindex.com/stanfordspezi/speziviews/1.0.0/documentation/speziviews/label).

Your implementation looks correct. If there is an issue, we can fix it together in these modules, and your app will automatically get the new behavior by updating your dependencies to the latest version. We will follow up soon! CC: @CS342/2024-instructors

**Reply by bryant-jimenez** [reply]
Sounds good, thanks Paul!

**Reply by PSchmiedmayer** [reply]
@bryant-jimenez It would be great if you can create an issue about this in our SpeziContacts repo so we can keep track of that: https://github.com/StanfordSpezi/SpeziContact/issues. I have seen this in an other app as well and I think we should fix this there.

**Reply by bryant-jimenez** [reply]
@PSchmiedmayer Sounds good, went ahead and created the issue!

**Reply by PSchmiedmayer** [reply]
Thank you!
